src/content/app/genome-browser/hooks/useGenomeBrowserTracks.ts

```diff
@@ -54,7 +54,9 @@ import type { FocusObject } from 'src/shared/types/focus-object/focusObjectTypes
 
 const useGenomeBrowserTracks = () => {
   const genomeId = useAppSelector(getBrowserActiveGenomeId) as string;
```

Comment:
`as string` is lying to us here; try removing it

Reply:
That's done